lib/response.js

```diff
@@ -163,7 +163,7 @@ res.send = function send(body) {
         }
       } else if (isChunkBlob) {
         if (!this.get('Content-Type')) {
-          this.type(chunk.type || 'bin');
+          this.setHeader('Content-Type', chunk.type || 'application/octet-stream');
```

Comment:
This would need an extra conditional as for no type, it needs to use `this.type('bin')` so it matches Buffer and uses any override the user set for generic type.